src/modules/commentPopup.js

```diff
@@ -0,0 +1,48 @@
+import closeSvg from '../assets/images/close.svg';
+
+const baseAPI = 'https://api.tvmaze.com/shows';
+
+const closed = () => {
+  const popWin = document.querySelector('.popup');
+  popWin.style.visibility = 'hidden';
+  return null;
```

Comment:
- [ ] Kindly ensure that the comment pop-up is designed accordance to the given template. 👍 

## Your Design
<img width="952" alt="Untitled" src="https://github.com/Kanyorok/Capstone-Grp_Kanban/assets/103386183/c7064627-a2d4-48cc-ab06-1a5f8e733ef3">

## Wireframe
<img width="148" alt="Untitled" src="https://github.com/Kanyorok/Capstone-Grp_Kanban/assets/103386183/1f2bd1f8-81b1-4001-82d4-7688270cd179">

Reply:
@Peter-Kibet - you didn't implement this requested change from the previous CR - as stated in the [project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md#:~:text=You%20should%20follow%20the%20layout%20of%20the%20wireframes%20provided).